data/upgrades.js

```diff
@@ -1800,6 +1810,18 @@
     "xws": "heavyscykinterceptor",
     "grants": [
       {
+        "type": "slot",
+        "name": "Cannon"
+      },
+      {
+        "type": "slot",
+        "name": "Torpedo"
+      },
+      {
+        "type": "slot",
+        "name": "Missile"
+      },
+      {
```

Comment:
Lets worry about that in a separate pr. Could you remove the `grants` from `Heavy Scyk` so I can merge this? :)